Services/CO.CDP.Organisation.WebApi/UseCase/UpdateOrganisationUseCase.cs

```diff
@@ -97,23 +97,22 @@ public async Task<bool> Execute((Guid organisationId, UpdateOrganisation updateO
 
                 break;
             case OrganisationUpdateType.AdditionalIdentifiers:
-                if (updateObject.AdditionalIdentifiers == null)
+                if (updateObject.AdditionalIdentifiers == null || !updateObject.AdditionalIdentifiers.Any())
                 {
                     throw new InvalidUpdateOrganisationCommand.MissingAdditionalIdentifiers();
                 }
 
                 foreach (var identifier in updateObject.AdditionalIdentifiers)
                 {
+                    await ValidateIdentifierIsNotKnownToUs(organisationRepository, identifier);
```

Comment:
No need to pass the `organisationRepository` here, as it's part of the class properties.